### PR TITLE
TropiCata: Tropical Bothanics I

### PR DESCRIPTION
**Pull request by Xaritscin**
#### Summary
Mods "Adds more tropical flora"

#### Purpose of change

Did a revision of all the current vanilla trees to check if there was space for inclusion. Except for Birches and Coffee Trees most of the basegame species are fit to stay in the tropical region but for now im just going to add more plant clutter instead of blacklisting species to keep it safe for players.

#### Describe the solution

added 2 new species of nut producing trees and 1 nut bush, all of them are using current vanilla nuts(for now, this may change later), along with 6 new species of non harvestable trees for all your woodcutting needs.

#### Describe alternatives you've considered

Leave as it is but i want to see more of that Latin American forestry in the map. That's the whole purpose of this mod.

#### Testing

Generated a new world, all the plants seem to be spawning. Albeit per map setting weights some may be rare occurences.

#### Additional context

Ficus is technically a resource tree but there's no figs or latex products in this game afaik. correct me if im wrong but for now they are just decorative.

### CODE REVIEW

**Comment by NetSysFire** [comment]
Out of curiosity, is there any region you define your mod to be in? Vanilla is in New England and yours is somewhere in South America? Any specific country or region?

**Comment by LyleSY** [comment]
Looks good but I think this should be frozen until the next experimental

File: data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json [hunk 1]
```diff
@@ -1,4 +1,301 @@
...
+        { "item": "splinter", "count": [ 10, 25 ] },
+        { "item": "leaves", "count": [ 50, 150 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_pachira_harvested",
+    "name": "chestnut tree",
+    "description": "A massive tree belonging to the 'Pachira' genus, also known as the Guyanan Chesnut tree.",
```
Comment (TheShadowFerret) [inline data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json:33]:
```suggestion
    "description": "A massive wetland tree belonging to the 'Pachira' genus, also known as the Guyanan Chesnut tree.",
```

File: data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json [hunk 2]
```diff
@@ -1,4 +1,301 @@
 [
+  {
+    "type": "terrain",
+    "id": "t_tree_pachira",
+    "name": "pachira tree",
+    "description": "A massive tree belonging to the 'Pachira' genus, also known as the Guyanan Chesnut Tree.  This species of tree grows on wetlands, if you look closely, you can see some unharvested chestnuts.",
```
Comment (TheShadowFerret) [inline data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json:6]:
```suggestion
    "description": "A massive wetland tree belonging to the 'Pachira' genus, also known as the Guyanan Chesnut Tree.  If you look closely, you can see some unharvested chestnuts.",
```

File: data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json [hunk 3]
```diff
@@ -1,4 +1,301 @@
...
+        { "item": "splinter", "count": [ 10, 25 ] },
+        { "item": "leaves", "count": [ 50, 150 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_parajubea",
+    "name": "parajubea",
+    "description": "A medium sized palm tree belonging to the 'Parajubea' genus.  There are only tree confirmed members of this genus, two of them endangered in Bolivia.",
```
Comment (TheShadowFerret) [inline data/mods/TropiCataclysm/furniture_and_terrain/tropical_terrain_flora.json:233]:
```suggestion
    "description": "A medium sized palm tree belonging to the 'Parajubea' genus.  There are only three confirmed members of this genus, two of them endangered in Bolivia.",
```
*Not sure if pun or typo*

**Comment by Xaritscin** [comment]
Got the notification this was going to get marked as stale so i upstreamed my repo just in case. shouldnt break anything on this PR i think.

**Pull request merged**